### PR TITLE
Force opencv to version 4.6.0 or higher

### PR DESCRIPTION
**Pull request by SamCWill**
# Force opencv to version 4.6.0 or higher
## What?
Force opencv to version 4.6.0 or higher
## Why?
Doug was running a dataset where some panels were not being detected. I ran the same dataset, and those panels were being detected. The difference? I was on opencv-contrib-python v4.6.0 and he was on v4.5.5. Might as well just lock it down.
## PR Checklist
- [X] Merged latest master
- [X] Updated version number
- [X] Version numbers match between package ``__version__`` and *pyproject.toml*
- [ ] All private git packages are at their newest version in both *pyproject.toml* and *environment.yml*
- [ ] All git package version numbers match between *pyproject.toml* and *environment.yml*
- [ ] dist/ImageryCorrector.exe and dist/GetCorrectionsCsv.exe have been rebuilt
## Breaking Changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,10 @@ install:
   - "perl Makefile.PL"
   - "sudo make install"
   - "cd .."
-  - "pip install poetry==1.1.15"
+  - 'curl -sSL https://install.python-poetry.org | python3 -'
+  - 'export PATH="/root/.local/bin:$PATH"'
+  - 'poetry self update 1.4.0'
+  - 'pip install "urllib3>=1.0.0,<2.0.0"'
```
Comment (elliott-imhoff) [inline .travis.yml:14]: Are you sure you need all this? In my experience all you need to do is the curl command to install poetry

**Review by elliott-imhoff — APPROVED** [review]
Just one comment

File: .travis.yml (hunk 1 above)
Comment (SamCWill) [reply .travis.yml:14]: I definitely need the urllib line. See previous failing builds. I like locking poetry to a specific version. I thought the PATH line was necessary but maybe not?

Comment (elliott-imhoff) [reply .travis.yml:14]: The urllib line was added before you updated the poetry install to use curl

Comment (SamCWill) [reply .travis.yml:14]: Hmm true. I'll try a build without it.

Comment (elliott-imhoff) [reply .travis.yml]: Yeah that urllib error occurs with old `pip install poetry` and then trying to install the package deps in the same environment. The curl command should be all that is needed: https://github.com/SenteraLLC/cvdb/blob/master/.travis.yml#L28-L29

Comment (elliott-imhoff) [reply .travis.yml]: You also should be able to pin a specific poetry version directly in the install command

Screen Shot 2023-06-21 at 4 43 03 PM

Comment (SamCWill) [reply .travis.yml]: Removing the urllib and PATH lines worked fine! Good call!

**Pull request merged**